packages/cadl-ranch-specs/http/azure/core/page/main.tsp

```diff
@@ -200,3 +201,93 @@ interface TwoModelsAsPageItem {
   @route("/second-item")
   listSecondItem is global.Azure.Core.Foundations.Operation<{}, global.Azure.Core.Page<SecondItem>>;
 }
+
+/**
+ * The interval
+ */
+union TimeInterval {
+  /**
+   * A string representing a custom interval.
+   */
+  string,
+
+  PT5S: "PT5S",
+  PT10S: "PT10S",
+  PT1M: "PT1M",
+  PT5M: "PT5M",
+  PT1H: "PT1H",
+}
+
+model QueryParameters {
+  /**
+   * The interval (i.e. timegrain) of the query.
+   */
+  @query
+  timeInterval?: TimeInterval;
+
+  /**
+   * The specific parameter name to be queried.
+   */
+  @query
+  parameterName?: string;
+
+  /**
+   * The namespace that the query is performed within.
+   */
+  @query
+  parameterNamespace: string;
+
+  /**
+   * The specific timespan for the query, defining the range of time for the data.
+   */
+  @query
+  timespan?: string;
+}
+
+@resource("values")
+@parentResource(Item)
+model ItemValues {
+  /**
+   * The array of values associated with the item.
+   */
+  @key
+  value: string[];
+}
+
+@resource("items")
+@parentResource(TestRuns)
+model Item {
+  @key
+  @path("name")
+  @visibility("query")
+  name: string;
+}
+
+/**
+ * Test run model
+ */
+@resource("test-runs")
+model TestRuns {
+  /**
+   * Unique identifier for the test run.
+   */
+  @key
+  @path
+  @visibility("read")
+  testRunId: string;
+}
+
+@scenario
+@scenarioDoc("""
+  List the item values.
+  """)
+interface AdditionalParameter {
+  /**
+   * List the item values.
+   */
+  @route("/items-traits")
+  listWithTraits is global.Azure.Core.ResourceList<
```

Comment:
I think we already have a very similar test in `azure/core/basic` https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/azure/core/basic/main.tsp#L178. Can you move that test instead into `azure/core/page`? Thank you!

Reply:
I'd against moving the test. List is one of the basic CRUD of any Azure resource. We already had it in basic for months. There is no point moving this test, just for the sake of moving...